src/modules/add-site/hooks/use-stepper.ts

```diff
@@ -18,32 +19,33 @@ interface StepperConfig {
 }
 
 interface StepperContext {
+	flow?: 'blueprint' | 'backup' | 'create';
 	steps: StepperStep[];
-	isVisible: boolean;
+	isVisible?: boolean;
 	actionButton?: {
 		label: string;
 		isVisible: boolean;
 	};
-	onSubmit: () => void;
-	canSubmit: boolean;
+	onSubmit?: () => void;
+	canSubmit?: boolean;
```

Comment:
Let's avoid making this elementos optional, and let's use a different type for `stepperConfig`.
```suggestion
	onSubmit: () => void;
	canSubmit: boolean;
```

See my comment below.